sql/core/benchmarks/DateTimeBenchmark-results.txt

```diff
@@ -414,3 +414,16 @@ to_date wholestage off                        1477 / 1479          0.7        14
 to_date wholestage on                         1468 / 1473          0.7        1468.2       1.0X
 
 
+================================================================================================
+Conversion from/to external types
+================================================================================================
+
+Java HotSpot(TM) 64-Bit Server VM 1.8.0_202-b08 on Mac OS X 10.14.4
+Intel(R) Core(TM) i7-4850HQ CPU @ 2.30GHz
+To/from java.sql.Timestamp:               Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
+------------------------------------------------------------------------------------------------------------------------
+From java.sql.Timestamp                             309            316           8         16.2          61.9       1.0X
+Collect longs                                      1410           2747        1158          3.5         282.0       0.2X
```

Comment:
Oh, could you rerun this in the more stable environment?
`Stdev` looks too high to use as a future reference.